site/run-letsencrypt.nginx.conf

```diff
@@ -2,5 +2,5 @@ server {
     listen 80 default_server;
     listen [::]:80 default_server;
     root /var/www/html;
-    server_name hail.is www.hail.is ci.hail.is ci2.hail.is scorecard.hail.is test.hail.is;
+    server_name hail.is www.hail.is ci.hail.is scorecard.hail.is upload.hail.is test.hail.is dev1.hail.is;
```

Comment:
who is dev1?

Reply:
Reserved for future use.  It's annoying to get new certs if you want to play with an idea.  Another alternative is to have dev-tools.hail.is/scorecard, etc.